src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -145,17 +198,77 @@ def repro(self, repro_timeout, repro_args):
     return engine.ReproduceResult(result.command, result.return_code,
                                   result.time_executed, result.output)
 
-  def fuzz(self,
-           fuzz_timeout,
-           additional_args,
-           unused_additional_args=None,
-           unused_extra_env=None):
+  def save_rawcover_output(self, pid: int):
+    """Find syzkaller port and write rawcover data to a file."""
+
+    port = find_syzkaller_port(pid)
+
```

Comment:
can delete lines 205-206 here (no need for the comment -- the code is obvious enough).